PhysicsTools/Utilities/interface/SimplifyTrigonometric.h

```diff
@@ -107,7 +105,12 @@ namespace funct {
   };
 
   TEMPL(N2T1) struct SimpifyS2C2Sum<n, m, A, false> {
-    static const int p = ::boost::mpl::if_c<(n < m), ::boost::mpl::int_<n>, ::boost::mpl::int_<m> >::type::value;
+    static const int p = []() {
```

Comment:
This can just be
```C++
static constexpr int p = n<m? n: m;
```

Reply:
Yeah, that's a lot more cleaner. Thanks.